### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by myelin0**
In this branch I create a full website for the Math magicians app, consisting of several pages and using the components I had already created.

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Use React basic styling.
- For every page, you should create a route.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,41 @@
 import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
+// import Calculator from './components/calculator';
```
Comment (Meltrust) [inline src/App.js]: - [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

File: src/Home.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// import { Link } from 'react-router-dom';
```
Comment (Meltrust) [inline src/Home.js]: - [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

File: src/components/calculator.css [hunk 3]
```diff
@@ -1,4 +1,4 @@
-*,
+/* *,
```
Comment (Meltrust) [inline src/components/calculator.css]: - [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

File: src/components/calculator.css [hunk 4]
```diff
@@ -8,30 +8,36 @@ body {
...
 .input {
   display: grid;
-  grid-template-columns: repeat(4, 6rem);
-  grid-template-rows: minmax(7rem, auto) repeat(5, 6rem);
-  height: 400px;
+  grid-template-columns: repeat(4, 25%);
+  grid-template-rows: minmax(3rem, auto) repeat(5, 3rem);
+  font-weight: 600;
+
+  /* height: 400px; */
```
Comment (Meltrust) [inline src/components/calculator.css]: - [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @myelin0,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Beautiful project✔️ 
- The app works ✔️ 
- Nice use of routes ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.css (hunk 4 above)
Comment (myelin0) [reply src/components/calculator.css]: done

File: src/components/calculator.css (hunk 3 above)
Comment (myelin0) [reply src/components/calculator.css]: done

File: src/Home.js (hunk 2 above)
Comment (myelin0) [reply src/Home.js]: done

File: src/App.js (hunk 1 above)
Comment (myelin0) [reply src/App.js]: done

**Review by Meltrust — APPROVED** [review]
Hi @myelin0,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**